curr_scraper/apps/api_caller/call_logic.py

```diff
@@ -0,0 +1,54 @@
+import json
+import os
+from datetime import datetime
+
+import pytz
+import requests
+
+from api_caller import models
+
+
+def call_api():
+    oe_url = os.environ.get('OPEN_EXCHANGE_URL')
+    oe_app_id = os.environ.get('OPEN_EXCHANGE_APP_ID')
+    oe_complete_url = f'{oe_url}/latest.json?app_id={oe_app_id}'
+    try:
+        response = requests.get(oe_complete_url)
+        json_data = json.loads(response.text)
+        base = json_data["base"]
+    except KeyError:
+        print("KeyError")
+    else:
+        return json_data
+
+
+def initialize_currency_table():
+    json_data = call_api()
+    rates = json_data["rates"]
+    curr_list = rates.keys()
+    for currency in curr_list:
+        cr = models.Currency(code=currency, description=currency)
```

Comment:
You are going to create new Currencies each time you run initialize command. 
Please use get_or_create: https://docs.djangoproject.com/en/4.0/ref/models/querysets/#get-or-create